### ISSUE TITLE
Change message on client when clicking trust

### ISSUE THREAD

**Issue by zarik5** [alvr-org/ALVR#278]
If you still see the "Click on trust" message after clicking on trust it's because of a connection problem. This should be clarified.
The message could become dynamic once the network rewrite is in place.

**Comment by ratozumbi** [comment]
I still see the trust message after clicking on trust. What can I do?
Steam VR opens, but no headest is found.
In this state (white screen), AVLR client has more frozen frames then rendered ones.

**Comment by zarik5** [comment]
@ratozumbi You're having a connection problem, that's out of the scope of this issue. Seek help on Discord or open another issue.

**Comment by zarik5** [comment]
The client can now display different messages depending on the connection state
